src/effects/VST/VSTEffect.cpp

```diff
@@ -2013,7 +2009,7 @@ int VSTInstanceBase::GetProcessLevel()
    return mProcessLevel;
 }
 
-void VSTInstanceBase::PowerOn()
+void VSTEffectWrapper::PowerOn()
```

Comment:
In step 5.4, think `PowerOn` and `PowerOff` should not move to Wrapper.  They are only for the Instance.

Power comes ON, only in an instance.  Note `vstEffInstance.PowerOn()` -- it's not done on This.

But there are calls to `PowerOff` done on the Effect.

The right thing is to change those.

Where `CloseUI` does `PowerOff`, do it instead to the Instance stored in the Validator.  Which is why `VSTEffect::mValidator` finds a use after all.

Als in `CloseUI`, reassign `mValidator` to null when done.

The `PowerOff()` in the unloading of the effect should just be deleted.